libs/ebpfnetsh/programs.cpp

```diff
@@ -110,7 +117,7 @@ handle_ebpf_add_program(
     bpf_attach_type attach_type = BPF_ATTACH_TYPE_UNSPEC;
     pinned_type_t pinned_type = PT_FIRST; // Like bpftool, we default to pin first.
     ebpf_execution_type_t execution = EBPF_EXECUTION_JIT;
-    LPWSTR interface_parameter = nullptr;
+    _Null_terminated_ wchar_t* interface_parameter = nullptr;
```

Comment:
My understanding was that SAL annotation doesn't apply to local variables.  Have you found anything that says this does something?